CodeStarterCamp_Week2/randomLottoNumber.swift

```diff
@@ -8,16 +8,37 @@
 import Foundation
 
 // 로또 당첨 번호를 생성하는 함수
-func makeLottoNumbers() -> Set<Int> {
+func makeLottoNumbers() {
     var lottoNumbers: Set<Int> = Set<Int>()
     var lottoNumber: Int
     while lottoNumbers.count < 6 {
         lottoNumber = Int.random(in: 1...45)
         lottoNumbers.insert(lottoNumber)
     }
-    return lottoNumbers
+    // 생성한 로또 당첨 번호를 저장
+    saveLottoNumbers(newLottoNumbers: lottoNumbers)
 }
 
+// 로또 당첨 번호를 저장하는 함수
+func saveLottoNumbers(newLottoNumbers: Set<Int>) {
+    let turn = lottoHistory.count
+    lottoHistory[String(turn+1)+"회차"] = newLottoNumbers
+}
+
+// n회차의 로또 당첨번호를 출력합니다.
+func findLottoNumbers(_ n: Int) {
+    let turn: String = String(n)+"회차"
```

Comment:
여기도 컨벤션 신경써주시면 좋겠습니다 ㅎㅎ
위에서는 type inference를 사용하고 있지만 이 부분에서는 type annotation을 사용하고 있네요.
```suggestion
    let turn = String(n) + "회차"
```

추가로, 여유가 되신다면 반복되는 코드를 줄여볼 수 있겠습니다.
turn에 할당할 String을 구해주는 기능의 함수로 분리해봅시다!